src/const/KeysWithoutPressEventDictionary.js

```diff
@@ -1,27 +0,0 @@
-/**
```

Comment:
🐥 Remove separate file previously used to record keys without keypress events - this is now just the list of non-printable keys.